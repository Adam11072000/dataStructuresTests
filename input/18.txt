init 2412
addImage -2
getHighestScoredLabel 3006 -7
addImage 1613
addImage -1
resetLabelScore 1613 829 0
addImage 3849
addImage -1
addImage -6
quit