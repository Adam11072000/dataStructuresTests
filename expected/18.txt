init done.
addImage: INVALID_INPUT
getHighestScoredLabel: INVALID_INPUT
addImage: SUCCESS
addImage: INVALID_INPUT
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
addImage: INVALID_INPUT
addImage: INVALID_INPUT
quit done.
